### PR TITLE
Request Reports

### PR DESCRIPTION
**Pull request by ludwiktrammer**

### CODE REVIEW

File: addons/bestja_requests/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            <field name="model_id" ref="model_bestja_requests_template"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                '|',
+                    ('organization.coordinator', '=', user.id),
+                    ('organization.projects.manager', '=', user.id),
+            ]</field>
+            <field name="perm_read" eval="True"/>
+            <field name="perm_write" eval="False"/>
+            <field name="perm_create" eval="True"/>
```
Comment (KrzysiekJ) [inline addons/bestja_requests/security/security.xml]: The name says “can be viewed”, but here we also grant the create permission.

File: addons/bestja_requests/security/security.xml [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            <field name="model_id" ref="model_bestja_requests_request"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                '|',
+                    ('project.manager', '=', user.id),
+                    ('project.organization.coordinator', '=', user.id),
+            ]</field>
+            <field name="perm_read" eval="True"/>
+            <field name="perm_write" eval="False"/>
+            <field name="perm_create" eval="True"/>
```
Comment (KrzysiekJ) [inline addons/bestja_requests/security/security.xml]: In the rule name we have “can see”, but we change also the `create` permission.

File: addons/bestja_requests/security/security.xml (hunk 1 above)
Comment (ludwiktrammer) [inline addons/bestja_requests/security/security.xml]: "Request templates can be viewed and *created* by coordinator and managers of organisations which *created* them." would sound stupid ;)

File: addons/bestja_requests/security/security.xml [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+            <field name="model_id" ref="model_bestja_requests_item"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                '|',
+                    ('request.project.manager', '=', user.id),
+                    ('request.project.organization.coordinator', '=', user.id),
+            ]</field>
+            <field name="perm_read" eval="True"/>
+            <field name="perm_write" eval="False"/>
+            <field name="perm_create" eval="True"/>
```
Comment (KrzysiekJ) [inline addons/bestja_requests/security/security.xml]: In the rule name we have “can see”, but we also modify the `create` permission.

**Pull request merged**